packages/itwinui-react/src/core/Toast/Toast.tsx

```diff
@@ -285,7 +285,13 @@ export const ToastPresentation = React.forwardRef((props, forwardedRef) => {
         {content}
       </Box>
       {link && (
-        <ButtonBase className='iui-toast-anchor' {...link} title={undefined}>
+        <ButtonBase
+          className='iui-anchor iui-toast-anchor'
```

Comment:
This should use `cx` for merging class names.

Also it should take into account `link.className`?

```jsx
<ButtonBase
  {...link}
  className={cx('iui-anchor', 'iui-toast-anchor', link.className)}
```

Reply:
Alright, will do.

Reply:
Done in this [commit.](https://github.com/iTwin/iTwinUI/pull/1915/commits/bf9f560bd5cdba19aa3422624f98b7086412affa)